lib/custom_html_renderer.rb

```diff
@@ -0,0 +1,7 @@
+require 'middleman-core/renderers/redcarpet'
+
+class CustomHTMLRenderer < Middleman::Renderers::MiddlemanRedcarpetHTML
```

Comment:
Is the idea that any other customisations would go in here too? The generic `CustomHTMLRenderer` name sounds like it might contain other stuff.